dockyard/common/information/information.py

```diff
@@ -0,0 +1,21 @@
+from dockyard.common import base, link
+from dockyard.common import utils
+
+class Information(object):
+    def __init__(self):
+        self.rest_client = base.RESTClient()
```

Comment:
Now this can be improved and methods made like dispatch_post_request etc can be used here.